packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -1096,10 +1108,18 @@ export class GarbageCollector implements IGarbageCollector {
 	 *
 	 * @param fromNodePath - The node from which the reference is added.
 	 * @param toNodePath - The node to which the reference is added.
+	 * @param timestampMs - The timestamp of the message that added the reference.
 	 * @param autorecovery - This reference is added artificially, for autorecovery. Used for logging.
 	 */
-	public addedOutboundReference(fromNodePath: string, toNodePath: string, autorecovery?: true) {
-		if (!this.shouldRunGC) {
+	public addedOutboundReference(
+		fromNodePath: string,
+		toNodePath: string,
+		timestampMs: number | undefined,
+		autorecovery?: true,
+	) {
+		// If there is no reference timestamp to work with, no ops have been processed after creation. If so, skip
+		// logging as nothing interesting would have happened worth logging.
```

Comment:
rephrase this, it's about way more tahn logging here.

Reply:
See other comments, ideally we can make timestampMs required.

Reply:
Made timestamp required.